docs/environment-variables.md

```diff
@@ -23,7 +23,7 @@ This document outlines environment variables that can be used to customize behav
 | `ARGO_AGENT_CPU_LIMIT`                   | `resource.Quantity` | `100m`                                                                                      | CPU resource limit for the agent.                                                                                                                                                                                                                                        |
 | `ARGO_AGENT_MEMORY_LIMIT`                | `resource.Quantity` | `256m`                                                                                      | Memory resource limit for the agent.                                                                                                                                                                                                                                     |
 | `ARGO_POD_STATUS_CAPTURE_FINALIZER`      | `bool`              | `false`                                                                                     | The finalizer blocks the deletion of pods until the controller captures their status.
-| `ARGO_TEMPLATE_WITH_INPUTS_PARAMETERS`      | `bool`              | `true`                                                                                     | Whether to keep inputs.parameters inside the ARGO_TEMPLATE environment variable of pods.
+| `ARGO_TEMPLATE_WITH_INPUTS_PARAMETERS`   | `bool`              | `true`                                                                                      | Whether to keep `inputs.parameters` inside the `ARGO_TEMPLATE` environment variable of pods.
```

Comment:
if we decide that we don't want to support anyone like me who was using ${ARGO_TEMPLATE} for input parameters anymore (per [this](https://github.com/argoproj/argo-workflows/pull/13761#issuecomment-2415589211) comment), then does it mean we should no longer have this environment variable? (I believe @tooptoop4 was trying to support backward compatibility for me and anyone like me, if there is anyone with this being an environment variable)

Reply:
Agree, `ARGO_TEMPLATE_WITH_INPUTS_PARAMETERS` removed.